node/src/types.ts

```diff
@@ -233,6 +238,7 @@ export interface Patch {
    * The id of the actor that generated this patch e.g. a web browser client, or file watcher
    */
   actor?: string
+  address: Slot[]
```

Comment:
@nokome: Is this field optional, or always defined, or defined but can be empty array?

Reply:
Optional. When defined could be an empty array but usually not.